docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-regression]]
+=== Predicting delays with {reganalysis} by using Kibana sample flight data
+
+
+The goal of <<dfa-regression,{reganalysis}>> is to find relationships among 
+different features of your data, then make predictions on the new data points 
+based on the previously found relationships. 
+
+Let's try to predict flight delays by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a model that allows us to predict flight delays based on information such as 
+weather and location of the destination and origin, flight distance and carrier. 
+We'd like to predict the number of minutes delayed for each flight. As it is a 
+continuous numeric variable, we'll use {reganalysis} to make the prediction.
+
+This dataset consists of sample data that has been manually created and it 
+contains inconsistencies. For example, a flight can be both delayed and 
+canceled. However, it is easy to set up in Kibana and the use case is 
+applicable, so it is suitable to be an example. Still, don't forget this is only 
+sample data, do not expect adequate results.
+
+We will create an {dfanalytics-job} to predict the minutes that flights are 
+delayed by. Each document in the dataset contains details for a single flight, 
+so this data is ready for analysis as it is already in a two-dimensional 
+entity-based data structure that we call a {dataframe}. Depending on your source 
+data, it may be required to first {ref}/transforms.html[transform] your source 
+data into an entity-centric index before you could setup and start the analysis.
+
+This is an example source document from the dataset:
+
+```
+{"_index":"kibana_sample_data_flights","_type":"_doc","_id":"S-JS1W0BJ7wufFIaPAHe","_version":1,"_seq_no":3356,"_primary_term":1,"found":true,"_source":{"FlightNum":"N32FE9T","DestCountry":"JP","OriginWeather":"Thunder & Lightning","OriginCityName":"Adelaide","AvgTicketPrice":499.08518599798685,"DistanceMiles":4802.864932998549,"FlightDelay":false,"DestWeather":"Sunny","Dest":"Chubu Centrair International Airport","FlightDelayType":"No Delay","OriginCountry":"AU","dayOfWeek":3,"DistanceKilometers":7729.461862731618,"timestamp":"2019-10-17T11:12:29","DestLocation":{"lat":"34.85839844","lon":"136.8049927"},"DestAirportID":"NGO","Carrier":"ES-Air","Cancelled":false,"FlightTimeMin":454.6742272195069,"Origin":"Adelaide International Airport","OriginLocation":{"lat":"-34.945","lon":"138.531006"},"DestRegion":"SE-BD","OriginAirportID":"ADL","OriginRegion":"SE-BD","DestCityName":"Tokoname","FlightTimeHour":7.577903786991782,"FlightDelayMin":0}}
```

Comment:
It was difficult to see this document in the preview due to long horiz scroll. Is it possible to view as line wrapped JSON?